code/datums/character_flaw/_character_flaw.dm

```diff
@@ -460,3 +461,30 @@ GLOBAL_LIST_INIT(character_flaws, list(
 
 /datum/charflaw/critweakness/on_mob_creation(mob/user)
 	ADD_TRAIT(user, TRAIT_CRITICAL_WEAKNESS, TRAIT_GENERIC)
+
+/datum/charflaw/disfigured
+	name = "Disfigured (2 TRI)"
+	desc = "In the past, I had barely managed to escape death's grasp. Though I still lyve, it is not without consequence. I am revolting to look upon, and misfortune seems to follow my every step."
+	var/nochekk = TRUE
+
+/datum/charflaw/disfigured/flaw_on_life(mob/user)
```

Comment:
`/datum/charflaw/proc/on_mob_creation(mob/user)`

or

`/datum/charflaw/proc/apply_post_equipment(mob/user)`

should be used here instead